### ISSUE TITLE
Friction 1.6.0-dev.8 Voicing Interviews

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/friction#208]
Interviews for the current state of the voicing feature in Friction were requested.

The following link was provided: https://phet-dev.colorado.edu/html/friction/1.6.0-dev.8/phet/friction_en_phet.html

Interview participants are requested to enable the voicing toggle before interacting with the simulation.

I will record a summary of the interviews in this issue.

**Comment by BLFiedler** [comment]
@Matthew-Moore240 @zepumph 

Two interviews were conducted on 7/29/2021 with sighted college students (P1 & P2) experienced on an iPad using the default Samantha voice. P1 leaves the voicing on the whole interview. P2 turns it off via the sidebar after ~2 minutes.

### General:
- Both participants self-reported the potential usefulness and completeness of the description if they could not see the simulation, but as redundant for them since they could see what was happening. Helpful for the changes they may have missed (or not actively looking at).
- P1 left the voicing on the entire time and was able to identify all of the sounds/sonifications, but did not note the change in the jiggling sonification. P2 claimed they did not notice many of the sonifications (jiggling, breaking, rubbing) until they turned off the voicing (and correctly mapped/applauded the sound design in general), and found the voicing distracting.
- Both participants noted that, by default, the voice was at a good rate.
- P1 explicitly enjoyed the voicing of menu text. 
- Both participants correctly identified the Preferences icon.

### Interaction:
- Both participants hesitated in their actions as they waited for the voicing to finish. Notably, P1 would jerk their hand back when the voicing would initiate to update the state of cooling/jiggling. P2 noted that **they would prefer one statement to indicate that the atoms jiggling is decreasing and temperature is decreasing, rather than repeated statements with updated temp/jiggling state.** 
- P1 noted that the voice interrupted itself frequently and worried they were missing important information.
- Both participants found the jiggling updates, break away notice, and temperature updates to be the most helpful statements in general. (BF: These are potential Context change checkbox statements I think)

### Toolbar:
- Neither participant interacted with this first, but both did on their own eventually. P1 found the overview and details helpful and descriptive (side note: giggled at the "warm up your hands" line). P2 found the sidebar buttons potentially helpful, but the overview was a bit too long. 
- P2 verbally approved of the ability to close the sidebar.

### Potential Bugs/Interaction Issues:
- While actively rubbing the books together, both participants received alerts to "Reset the sim to make more observations". I believe this happened because all of the atoms had broken away, but it was a confusing alert while they were still moving the books.
- While rubbing the books together, with finger in the middle upper portion of the screen, somehow P2 disabled Audio entirely. It's possible they bump the Mute All icon with the lower part of their hand, but I'm not sure.
- P1 noted that while other alerts interrupt themselves, Reset All does not interrupt and the voice continues to describe state of the system before the Reset All alert occurs. (BF: I believe this is confusing because the voice continues to describe the state of a system that no longer exists, along with a long delay in voicing the action they did (hitting the reset button)).

**Comment by BLFiedler** [comment]
I have the potential to interview further on this next week Tuesday. Please let me know if there are any specific questions you have that I can address based on the interviews that have already occurred, or that can be addressed with a quick additional interview.

**Comment by terracoda** [comment]
The "Reset sim to make more observations," is indeed one of those hints specifically design for non-visual access. Also, "Move down to rub harder" was designed specifically for alternative input users like blind users.

Technically speaking in Interactive Description, the reset-sim-hint happens when most of the atoms have evaporated, but not all. The timing of the delivery could be re-considered.

I think both these responses are categorized as hint responses, so once the checkboxes are implemented they can be enabled and disabled.

Hopefully, that's helpful information for @Matthew-Moore240 and @zepumph

**Comment by terracoda** [comment]
Maybe the Reset All button response can be assertive? That's likely easily fixable.

**Comment by zepumph** [comment]
Ready to close by the team.
